IoTCoreDefaultApp/IoTCoreDefaultApp/IoTCoreDefaultApp.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>IoTCoreDefaultApp</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10545.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.14993.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.14993.0</TargetPlatformMinVersion>
```

Comment:
Lets evaluate a conditional include here - only include 14993 if that SDK is installed